lib/cache/sandbox.ex

```diff
@@ -22,7 +22,7 @@ defmodule Cache.Sandbox do
   def child_spec({cache_name, opts}) do
     %{
       id: "#{cache_name}_anana_cache_agent",
-      start: {Cache.Agent, :start_link, [Keyword.put(opts, :name, cache_name)]}
+      start: {__MODULE__, :start_link, [Keyword.put(opts, :name, cache_name)]}
```

Comment:
@MikaAK I think this was what's intended here, but just flagging it up for you in case I'm missing something